package.json

```diff
@@ -15,6 +17,11 @@
   "author": "mieszko4",
   "license": "MIT",
   "peerDependencies": {
-    "react-native": ">0.41.2"
+    "react-native": ">0.60.0"
```

Comment:
this drops dependency of anything lower than 0.60.
that should be reflected in the docs.

Reply:
It doesn't drop. It will show warnings for old react-native during library install. 
If someone still using old react-native, anyway they should be forced to update.